CHANGELOG.md

```diff
@@ -1,4 +1,26 @@
 # Changelog
+## 1.32.5 (2024-11-24)
+### Breaking
+* Removed PA134 from the old validate, the validate ensured the pack has exactly 1 category and that the category is a valid category. [#4673](https://github.com/demisto/demisto-sdk/pull/4673)
+
+### Feature
+* Modified PA103 in the new validate (validate pack categories), the validation will now allow multiple categories for a single pack. [#4673](https://github.com/demisto/demisto-sdk/pull/4673)
+* Modified the configuration section generated by the generate-docs command by removing navigation steps and making the code more generic to support various Cortex platforms. [#4585](https://github.com/demisto/demisto-sdk/pull/4585)
+* Added support for non-default registry ports when using custom docker images. i.e: 'registry:5000/repository/image:main' [#4674](https://github.com/demisto/demisto-sdk/pull/4674)
+* Adds support for content_status.json to be used in the content pipeline [#4656](https://github.com/demisto/demisto-sdk/pull/4656)
+
+### Fix
+* Fixed an issue where `pre-commit` inadvertently skipped all files when executed with the `--all-files` flag. [#4668](https://github.com/demisto/demisto-sdk/pull/4668)
+* Fixed an issue where SDK commands would fail in some system setups. [#4666](https://github.com/demisto/demisto-sdk/pull/4666)
```

Comment:
```suggestion
* Fixed an issue where demisto-sdk commands would fail in some system setups. [#4666](https://github.com/demisto/demisto-sdk/pull/4666)
```